docs/howtos.rst

```diff
@@ -125,7 +125,7 @@ Define and use other global arguments
 
 To define more automatic global arguments than the ones provided automatically by ``Dispatcher`` (see :ref:`this explanation <expl_global_args>` for more information), use the ``GlobalArgument`` object to create all you need and pass them to the ``Dispatcher`` at instatiaton time.
 
-Check `its reference <craft_cli.dispatcher.html#craft_cli.dispatcher.GlobalArgument>`_ for more information about the parameters needed, but it's very straightforward to create these objects. E.g.::
+Check :class:`its reference <craft_cli.dispatcher.GlobalArgument>` for more information about the parameters needed, but it's very straightforward to create these objects. E.g.::
```

Comment:
I would directly expose the class name here:

```suggestion
Check :class:`craft_cli.dispatcher.GlobalArgument` for more information about the parameters needed, but it's very straightforward to create these objects. E.g.::
```

Reply:
If not, use as you used Dispatcher below

Reply:
Improved.